management/src/main/java/org/kaazing/gateway/management/config/ServiceConfigurationBeanImpl.java

```diff
@@ -326,6 +336,16 @@ public String getConnectOptions() {
                         }
                     }
 
+                    Long tcpHandshakeTimeout = (Long) connectOptions.remove("tcp.handshake.timeout");
+                    if (tcpHandshakeTimeout != null) {
+                        jsonOptions.put("tcp.handshake.timeout", tcpHandshakeTimeout);
+                    }
+
+                    Long httpHandshakeTimeout = (Long) connectOptions.remove("http.handshake.timeout");
+                    if (httpHandshakeTimeout != null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaazing.wan:9000/coding_rules#rule_key=squid%3AS134)